components/forms/Donate/Donateguild.tsx

```diff
@@ -28,7 +28,7 @@ export default function DonateGuild() {
 
   console.log(input)
```

Comment:
should we remove this log?

Reply:
It was there before. We can get the TS conversion in then make other improvements later.